apps/src/weblab/WebLab.js

```diff
@@ -288,6 +377,11 @@ WebLab.prototype.onProjectChanged = function () {
 // Called by Bramble host to set our reference to its interfaces
 WebLab.prototype.setBrambleHost = function (obj) {
   this.brambleHost = obj;
+  this.brambleHost.onBrambleReady(() => {
+    if (this.hideSource) {
+      this.brambleHost.enableFullscreenPreview();
+    }
+  });
```

Comment:
While this works, it visibly switches to fullscreen preview after rendering in normal mode. A Bramble change appears to be required to allow it to initialize in "fullscreen preview" mode.